sys/libraries/deployment_settings.php

```diff
@@ -0,0 +1,22 @@
+<?php
+class deployment_settings extends Library {
+
+    // Uncomment the next line to deactivate Missing Person Module
+    // private $missing_person = false;
```

Comment:
How would we include it then?? simpy like "include_once config.php; $settings = new deployment_settings();" ????